tests/mbed/host/ocalls.c

```diff
@@ -1,8 +1,11 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#pragma warning(disable : 4996)
```

Comment:
Can this line be put inside a push/pop surrounding whatever code is affected?

Reply:
It applies to all the four functions so I'll just keep it here.